### PR TITLE
Spec: Syntax changes.

### PR DESCRIPTION
**Pull request by sjrd**
[skip ci]

### CODE REVIEW

File: docs/_spec/01-lexical-syntax.md [hunk 1]
```diff
@@ -121,12 +119,35 @@ type      val       var       while     with      yield
 
 Additionally, the following soft keywords are reserved only in some situations.
 
-´\color{red}{\text{TODO SCALA3: Port soft-modifier.md and link it here.}}´
-
 ```
-as  derives  end  extension  infix  inline  opaque  open  transparent  using  |  *  +  -
+as      derives      end       extension infix     inline    opaque
```
Comment (bishabosha) [inline docs/_spec/01-lexical-syntax.md:123]: seems uncomfortably close
```suggestion
as      derives      end       extension    infix     inline    opaque
```

File: docs/_spec/04-basic-declarations-and-definitions.md [hunk 2]
```diff
@@ -729,64 +735,86 @@ completely.  It is an error if the types of two alternatives ´T_i´ and
...
-Import selectors work in the same way for type and term members.
-For instance, an import clause `import ´p´.{´x´ => ´y\,´}` renames the term
-name `´p´.´x´` to the term name ´y´ and the type name `´p´.´x´` to the type name ´y´.
+Non-given selectors make available each importable member `´p´.´x_i´` under the unqualified name ´y_i´.
+In other words, every import selector `´x_i´ as ´y_i´` renames `´p´.´x_i´` to ´y_i´.
+When `as ´y_i´` is omitted, ´y_i´ is assumed to be ´x_i´.
+If a final wildcard `‘*’` is present, all non-`given` importable members ´z´ of ´p´ other than `´x_1, ..., x_n, y_1, ..., y_n´` are also made available under their own unqualified names.
+
+Non-given import selectors work in the same way for type and term members.
+For instance, an import clause `import ´p´.{´x´ => ´y´}` renames the term name `´p´.´x´` to the term name ´y´ and the type name `´p´.´x´` to the type name ´y´.
```
Comment (bishabosha) [inline docs/_spec/04-basic-declarations-and-definitions.md:778]:
```suggestion
For instance, an import clause `import ´p´.´x´ as ´y´` renames the term name `´p´.´x´` to the term name ´y´ and the type name `´p´.´x´` to the type name ´y´.
```

**Review by bishabosha — APPROVED** [review]
lgtm except some style points

File: docs/_spec/04-basic-declarations-and-definitions.md [hunk 3]
```diff
@@ -729,64 +735,86 @@ completely.  It is an error if the types of two alternatives ´T_i´ and
...
-```ebnf
-Import          ::= ‘import’ ImportExpr {‘,’ ImportExpr}
-ImportExpr      ::= StableId ‘.’ (id | ‘_’ | ImportSelectors)
-ImportSelectors ::= ‘{’ {ImportSelector ‘,’}
-                    (ImportSelector | ‘_’) ‘}’
-ImportSelector  ::= id [‘=>’ id | ‘=>’ ‘_’]
 ```
+Import            ::=  ‘import’ ImportExpr {‘,’ ImportExpr}
+ImportExpr        ::= SimpleRef {‘.’ id} ‘.’ ImportSpecifier
+                    | SimpleRef `as` id
```
[the remark this answers is not in the document]
Comment (sjrd) [reply docs/_spec/04-basic-declarations-and-definitions.md:741]: Yes, that's normal. `SimpleRef` is a syntax-level non-terminal. The lexical syntax page only shows lexer-level tokens.

**Pull request merged**